### ISSUE TITLE
Version overridden when also referencing asciidoctor-pdf

### ISSUE THREAD

**Issue by mrotteveel** [asciidoctor/asciidoctor-gradle-plugin#239]
When only using the html5 backend with asciidoctor-gradle-plugin 1.5.7, asciidoctorj 1.5.6 is used (the generated html has `<meta name="generator" content="Asciidoctor 1.5.6.1">`). 

If I reference `org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.16` because I also want to generate PDF, then this is overridden and 1.5.5 is used (`<meta name="generator" content="Asciidoctor 1.5.5">`), even when explicitly specifying the version in the asciidoctorj tag. This probably happens because `asciidoctorj-pdf:1.5.0-alpha.16` explicitly depends on asciidoctorj 1.5.5.

I'd expect the version declared in the asciidoctorj task to take precedence.

build.gradle:

```
defaultTasks 'clean', 'asciidoctor'

buildscript {
    repositories {
        jcenter()
    }

    dependencies {
        classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.7'
        //classpath 'org.asciidoctor:asciidoctorj:1.5.6'
        classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.16'
    }
}

apply plugin: 'org.asciidoctor.convert'

repositories {
    jcenter()
}

asciidoctorj {
    // To override default version of asciidoctorj use: version = '1.x.y'
    version = '1.5.6'
}

asciidoctor {
    sourceDir 'src/docs'
    backends=['html5', 'pdf']
}
```

As a workaround, I need to explicitly add a dependency on `org.asciidoctor:asciidoctorj:1.5.6` (uncomment it in the build.gradle above).

**Comment by ysb33r** [comment]
Yuo can also do 
```groovy
classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.16', {
    exclude module : 'org.asciidoctorj'
}
```

**Comment by mrotteveel** [comment]
@ysb33r The workaround works, but I consider the behavior itself a bug.

**Comment by ysb33r** [comment]
This is resolved in the 2.0-alpha.3 release, where one can control the versions in a simplified way, but still with fine granularity.

**Comment by mrotteveel** [comment]
Ok, thanks!
